pkgs/applications/video/obs-studio/default.nix

```diff
@@ -39,6 +40,13 @@ in stdenv.mkDerivation rec {
     sha256 = "0ri9qkqk3h71b1a5bwpjzqdr21bbmfqbykg48l779d20zln23n1i";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/obsproject/obs-studio/pull/1557.diff";
+      sha256 = "0xq3sh1g8j58mpa4ryi9agqqpr41prrzagxmqafkbgfv6fjv3h0c";
```

Comment:
I'm guessing the patch has changed : 

```
fixed-output derivation produced path '/nix/store/ij8k22jgavc1nzjvj4hy72av4iwlfswg-1557.diff' with sha256 hash '162fnkxh2wyn6wrrm1kzv7c2mn96kx35vlmk2qwn1nqlifbpsfyq' instead of the expected hash '0xq3sh1g8j58mpa4ryi9agqqpr41prrzagxmqafkbgfv6fjv3h0c'
```

Reply:
This is strange:
```
➜ nix-prefetch-url https://patch-diff.githubusercontent.com/raw/obsproject/obs-studio/pull/1557.diffpath is '/nix/store/raqc9k54zjgmxn7lwqmn0ycyhnjsr179-1557.diff'
0xq3sh1g8j58mpa4ryi9agqqpr41prrzagxmqafkbgfv6fjv3h0c
```

Reply:
@nyanloutre :thinking: interesting :sweat_smile: